### PR TITLE
Fix Print HEC bug under Py3

### PR DESCRIPTION
**Pull request by nabobalis**
Probably. #2043 Who really knows at this time.

I personally do not like the style of the fix. That could probably be improved.

### CODE REVIEW

**Comment by nabobalis** [comment]
The tests passed locally. Clearly there are something and they failed.

File: sunpy/net/helio/hec.py [hunk 1]
```diff
@@ -70,6 +70,8 @@ def suds_unwrapper(wrapped_data):
         </RESOURCE>
      </VOTABLE>
     """
+    if six.PY3 and not isinstance(wrapped_data,str):
+        wrapped_data = wrapped_data.decode("utf-8")
```
Comment (nabobalis) [inline sunpy/net/helio/hec.py]: SO UGLY.

File: sunpy/net/helio/hec.py [hunk 2]
```diff
@@ -70,6 +70,8 @@ def suds_unwrapper(wrapped_data):
         </RESOURCE>
      </VOTABLE>
     """
+    if six.PY3 and not isinstance(wrapped_data,str):
```
Comment (Cadair) [inline sunpy/net/helio/hec.py]: space after comma

File: sunpy/net/helio/hec.py (hunk 1 above)
Comment (Cadair) [inline sunpy/net/helio/hec.py]: explicit is better than implicit.

**Pull request merged**